mirage/route-handlers/crates.js

```diff
@@ -3,6 +3,10 @@ import { Response } from 'miragejs';
 import { getSession } from '../utils/session';
 import { compareIsoDates, compareStrings, notFound, pageParams, releaseTracks } from './-utils';
 
+function toCanonicalName(name) {
+  return name.toLowerCase().replace(/_/g, '-');
+}
```

Comment:
🙋 The canonical implementation in the backend(database) actually replaces `-` with `_`, but here we replace `_` with `-`. Is this difference intentional?

Reply:
> Is this difference intentional?

no, it's not intentional. I guess ultimately it doesn't really matter which way around we do it, but I'll adjust it so that it matches.